### DISCUSSION TITLE
A suggestion

### DISCUSSION THREAD

**Discussion by Kokie1111** [Vee-Gae/vee#13] [Ideas]
make a discord bot that says "vee gæ" everytime vee types something in discord

**Comment by sinjs** [comment]
Good Idea!

**Comment by Kokie1111** [comment]
> Good Idea!

dm me on discord when its done

**Comment by sinjs** [comment]
> dm me on discord when its done

Gonna do that, should I put the bot open source too? also I'm gonna be bombed with ratelimits

**Comment by Kokie1111** [comment]
> > dm me on discord when its done
> 
> Gonna do that, should I put the bot open source too? also I'm gonna be bombed with ratelimits

sure but if you get ratelimited make it happen every 30 seconds if vee types or it says vee gae in vee's dm's every half hour
